components/avatar/overview.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Overview
+page_title: Avatar Overview
+description: Discover the Blazor Avatar and explore the examples.
+slug: avatar-overview
+tags: telerik,blazor,avatar,overview
+published: True
+position: 0
+---
+
+# Blazor Avatar Overview
+
+The <a href="https://www.telerik.com/blazor-ui/avatar" target="_blank">Blazor Avatar component</a> is typically used to display images, icons or initials representing people or other entities.
+
+It also gives you the flexibility to customize its size, color, fill mode and more.
+
+## Creating Avatar
+
+1. Use the `TelerikAvatar` tag to add the component to your razor page.
+
+1. Declare your desired content inside the `TelerikAvatar` tag - text, icon or image.
```

Comment:
We can add the actual technical term for the space between the opening and closing tag - `ChildContent`.

Reply:
Not sure how recognized is `ChildContent` by devs, but I agree that I stumbled upon this step and my first thought was "how"? Probably I expected a property and didn't understand that we are suggesting nested content inside the component tag.

Reply:
Can you please clarify to make sure we are on teh same page? Your suggestion is

```suggestion
1. Declare your desired `ChildContent` inside the `TelerikAvatar` tag - text, icon or image.
```